.github/workflows/swagger.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Swagger Specs
+
+on:
+  push:
+    branches:
+      - master
+      - AM-2793_Publish_Swagger_Docs
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
```

Comment:
Should we switch to: https://github.com/hmcts/spring-boot-template/blob/master/.github/workflows/publish-openapi.yaml to cover Java 17 (i.e. if Java 11 is to be retired from pipeline soon).